Gemfile.jruby-1.9.lock

```diff
@@ -462,7 +462,8 @@ GEM
       logstash-core-plugin-api (>= 1.60, <= 2.99)
       redis
       stud
-    logstash-output-s3 (3.2.0)
+    logstash-output-s3 (4.0.0)
```

Comment:
I'll publish an s3 output 4.0.1, that add a big more debug when we check buckets permission.
Since we did a complete rewrite I would like to get it in, that could help future bug report.